themes/openy_themes/openy_rose/openy_rose.theme

```diff
@@ -104,3 +105,19 @@ function openy_rose_preprocess_field__node__field_location_directions(&$variable
   // Change link title to Directions.
   $variables['items'][0]['content']['#title'] = t('Directions');
 }
+
+/**
+ * Prepares variables for entity embed container templates.
+ *
+ * Default template: entity-embed-container.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - element: An associative array containing the properties of the element.
+ *     Properties used: #attributes, #children.
+ */
+function openy_rose_preprocess_entity_embed_container(&$variables) {
```

Comment:
Is it possible to move this preprocess to the openy_media? So it could be part of package.